lib/dark_worlds_server_web/websockets/play_websocket.ex

```diff
@@ -51,16 +47,17 @@ defmodule DarkWorldsServerWeb.PlayWebSocket do
 
   def websocket_init(%{
         game_id: game_id,
-        player_id: _player_id,
         client_id: client_id,
-        player_name: player_name
+        player_name: player_name,
+        selected_character: selected_character
       }) do
     runner_pid = Communication.external_id_to_pid(game_id)
 
     with :ok <- Phoenix.PubSub.subscribe(DarkWorldsServer.PubSub, "game_play_#{game_id}"),
          true <- runner_pid in RunnerSupervisor.list_runners_pids(),
          # String.to_integer(player_id) should be client_id
-         {:ok, player_id} <- Runner.join(runner_pid, client_id, "muflus") do
+
+         {:ok, player_id} <- Runner.join(runner_pid, client_id, selected_character) do
```

Comment:
Like Manuel said, the backend doesn't allow a match to start if the selected character doesn't exist, the only thing that is left is to handle that response from the backend in the client, I believe we could leave it for other issue since the error existed even before these changes, but I can see an argument being made for making the change in [this client PR](https://github.com/lambdaclass/curse_of_myrra/pull/1221).

Reply:
I think handling that should be done on the server. Despite that, I agree that we don't need to do this in this PR